src/plugins/intel_cpu/src/nodes/executors/convert_list.hpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "executor.hpp"
+
+#include "convert.hpp"
+#if defined(OV_CPU_WITH_ACL)
+#include "acl/acl_convert.hpp"
+#endif
+
+#include "common/ref_convert.hpp"
+
+#include "onednn/iml_type_mapper.h"
+#include "common/primitive_cache.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+struct ConvertExecutorDesc {
+    ExecutorType executorType;
+    ConvertExecutorBuilderCPtr builder;
+};
+
+const std::vector<ConvertExecutorDesc>& getConvertExecutorsList();
+
+class ConvertExecutorFactory : public ExecutorFactory {
+public:
+    ConvertExecutorFactory(const ConvertParams& convertParams,
+                             const std::vector<MemoryDescPtr>& srcDescs,
+                             const std::vector<MemoryDescPtr>& dstDescs,
+                             const ExecutorContext::CPtr context) : ExecutorFactory(context) {
+        for (auto& desc : getConvertExecutorsList()) {
+            if (desc.builder->isSupported(convertParams, srcDescs, dstDescs)) {
+                supportedDescs.push_back(desc);
+            }
+        }
+    }
+
+    ~ConvertExecutorFactory() = default;
+    virtual ConvertExecutorPtr makeExecutor(const ConvertParams& convertParams,
```

Comment:
It doesn't make much sense to keep this method implementation in header.

Reply:
moved